docs/helm/examples/with-elasticsearch-auth.md

```diff
@@ -0,0 +1,77 @@
+---
+title: ACS with local elasticsearch cluster with auth enabled
+parent: Examples
+grand_parent: Helm
+---
+
+# Deploying Elasticsearch with Kibana and Authentication Enabled
+
+This guide demonstrates how to deploy Elasticsearch with Kibana and
+authentication enabled using the Alfresco ACS Deployment repository.  
+
+> :warning: This example demonstrates how to use an embedded Elasticsearch cluster
+> with authentication enabled. However, it is intended for demonstration purposes
+> only and is **not recommended for production use**. We strongly advise against using
+> an embedded Elasticsearch cluster in production environments. This example is
+> provided solely for cases where the embedded setup is specifically needed, such
+> as for testing or development scenarios.
+
+## Prerequisites
+
+Ensure you have the following:
+
+- Helm installed on your system.
+- Kubernetes cluster configured and running.
+
+## Steps to Deploy
+
+### 1. Create a Secret
+
+First, create a Kubernetes secret containing the credentials for Elasticsearch
+and Kibana. Customize the values as needed for your setup.
+
+```bash
+ELASTIC_USERNAME="elastic" ELASTIC_PASSWORD="alfresco" KIBANA_PASSWORD="alfrescokibana" && \
+kubectl create secret generic elastic-search-secret \
+    --namespace=default \
+    --from-literal=elastic-user=$ELASTIC_USERNAME \
+    --from-literal=elasticsearch-password=$ELASTIC_PASSWORD \
+    --from-literal=kibana-password=$KIBANA_PASSWORD \
+    --from-literal=AUDIT_ELASTICSEARCH_USERNAME=$ELASTIC_USERNAME \
+    --from-literal=AUDIT_ELASTICSEARCH_PASSWORD=$ELASTIC_PASSWORD \
+    --from-literal=SEARCH_USERNAME=$ELASTIC_USERNAME \
+    --from-literal=SEARCH_PASSWORD=$ELASTIC_PASSWORD
```

Comment:
see `--from-env-file` which should prevent user leaking credentials in its history